### DISCUSSION TITLE
react 组件库搭建 - Alan|前端博客

### DISCUSSION THREAD

**Discussion by giscus** [3Alan/site#10] [Announcements]
# react 组件库搭建 - Alan|前端博客

react组件库的搭建流程，包括打包发布单元测试

https://www.alanwang.site/posts/react-components/react-%E7%BB%84%E4%BB%B6%E5%BA%93%E6%90%AD%E5%BB%BA

**Comment by dlzdd** [comment]
老哥，我昨天晚上听你的用docusaurus然后配合vercel然后配置域名已经搭建起来了，然后就是你首页的那个字体一个个出来的有什么组件嘛？我觉得很好看

**Reply by 3Alan** [reply]
使用这个库开发的
https://github.com/mattboldt/typed.js/
具体实现可以参考我的网站源码
https://github.com/3Alan/site
